lib/src/df.dart

```diff
@@ -31,12 +29,15 @@ class DataFrame {
   // ********* data **********
 
   /// An iterable of rows data
-  Iterable<Map<String, dynamic>> get rows => _iterRows();
```

Comment:
I've replaced `dynamic` with `Object` wherever it appears. Dynamic allows you to call any method without a compile time error. If the runtime type doesn't have that method, you'll get a runtime error. Object does not allow you to call arbitrary methods and forces you to explicitly cast the variable first. Additionally, `dynamic` is implicitly nullable whereas `Object` is non-nullable by default and must be declared nullable.

Because of the above two reasons, code using `Object` is safer and easier to read so I've switched to it.